### DISCUSSION TITLE
【问题反馈】第6课tvm.tir.function.PrimFunc没有show()方法

### DISCUSSION THREAD

**Discussion by Relph1119** [mlc-ai/mlc-zh#88] [Q&A] [answered]
**使用环境：**
Python 3.7.9
mlc-ai-nightly 0.9.dev1663+gc73f34568

**问题描述：**
![image](https://user-images.githubusercontent.com/15226799/182010552-5eb41222-a5ba-4a5b-b81b-796f33d912d1.png)
该图中表示没有这个show()，查看源码后，也发现没有这个函数
![image](https://user-images.githubusercontent.com/15226799/182010572-5d4d109c-e394-4e4a-9889-a8a5c8155a7c.png)

于是我自己实现了一个show()方法：
```python
    def show(self):
        import IPython

        def code2html(code):
            """Helper function to use pygments to turn the code string into highlighted html."""
            import pygments
            from pygments.lexers import Python3Lexer
            from pygments.formatters import HtmlFormatter
            formatter = HtmlFormatter()
            html = pygments.highlight(code, Python3Lexer(), formatter)
            return "<style>%s</style>%s\n" % (formatter.get_style_defs(".highlight"), html)

        return IPython.display.HTML(code2html(self.script()))
```
可以正常运行了
![image](https://user-images.githubusercontent.com/15226799/182010904-81cb5f83-3861-496d-9dd5-4ecaaebaabc2.png)

**Comment by Hzfengsy** [comment]
建议每节课前更新mlc的wheel，现在最新版应该是`0.9.dev1955+g495ac6010`

**Reply by Relph1119** [reply]
Window环境 Python3.7 的mlc-ai-nightly还没有对应的0.9.dev1955+g495ac6010版本
另：我将python环境升至python3.10，同样会报错，见[https://github.com/mlc-ai/mlc-zh/discussions/13](url)

**Comment by Relph1119** [comment, marked as answer]
已将mlc-ai-nightly升级到0.9.dev1957+gc164d2206
